appshell/cefclient.cpp

```diff
@@ -95,7 +96,21 @@ void AppGetSettings(CefSettings& settings, CefRefPtr<CefCommandLine> command_lin
       command_line->GetSwitchValue(client::switches::kJavascriptFlags);
     
   // Enable dev tools
-  settings.remote_debugging_port = REMOTE_DEBUGGING_PORT;
+  CefString debugger_port = command_line->GetSwitchValue("remote-debugging-port");
+  if (!debugger_port.empty()) {
+    int port = atoi(debugger_port.ToString().c_str());
```

Comment:
using atom seems acceptable in this case since the string is coming from the CommandLine handler. Better would be to use strtol.

Reply:
Changed parsing to use `strtol`.